client/app/visualizations/cohort/cohort-editor.html

```diff
@@ -1,8 +1,53 @@
-<div class="form-group">
+<ul class="tab-nav">
+  <li ng-class="{active: currentTab == 'options'}">
+    <a ng-click="setCurrentTab('options')">Options</a>
+  </li>
+  <li ng-class="{active: currentTab == 'columns'}">
+    <a ng-click="setCurrentTab('columns')">Columns</a>
+  </li>
+</ul>
+
+<div ng-if="currentTab == 'options'" class="m-t-10 m-b-10">
+  <div class="form-group">
     <label class="control-label">Time Interval</label>
     <select class="form-control" ng-model="visualization.options.timeInterval">
       <option value="daily">Daily</option>
       <option value="weekly">Weekly</option>
       <option value="monthly">Monthly</option>
     </select>
+  </div>
+
+  <div class="form-group">
+    <label class="control-label">Mode</label>
+    <select class="form-control" ng-model="visualization.options.mode">
+      <option value="diagonal">Diagonal</option>
+      <option value="simple">Simple</option>
+    </select>
+  </div>
+</div>
+
+<div ng-if="currentTab == 'columns'" class="m-t-10 m-b-10">
+  <div class="form-group">
+    <label class="control-label">Date</label>
+    <select class="form-control" ng-model="visualization.options.dateColumn"
+      ng-options="column for column in columnNames"></select>
+  </div>
+
+  <div class="form-group">
+    <label class="control-label">Day Number</label>
```

Comment:
`Stage`? 

@kocsmy any idea for a friendly name here?

Reply:
I'm also not sure about name for this, because, actually, it's "Day number" only when time interval is "daily", and for other values it matches time interval - so it may be "Week number"/"Month number". In the code it's called "day number" too, but obviously it's not good too. If you have ideas for better name - I'll be happy to apply it